packages/animations/lib/src/page_transition_switcher.dart

```diff
@@ -203,14 +235,70 @@ class PageTransitionSwitcher extends StatefulWidget {
   /// The child provided to the transitionBuilder may be null.
   final PageTransitionSwitcherTransitionBuilder transitionBuilder;
 
+  /// A function that wraps all of the children that are transitioning out, and
+  /// the [child] that's transitioning in, with a widget that lays all of them
+  /// out. This is called every time this widget is built. The function must not
+  /// return null.
+  ///
+  /// The default [PageTransitionSwitcherLayoutBuilder] used is
+  /// [defaultLayoutBuilder].
+  ///
+  /// The following example shows a [layoutBuilder] that places all children inside
+  /// a [Column] that sizes its height depending on the heights of active entries.
+  ///
+  /// ```dart
+  /// PageTransitionSwitcher(
+  ///   duration: const Duration(milliseconds: 100),
+  ///   child: Container(color: Colors.red),
+  ///   layoutBuilder: (
+  ///     Widget currentChild,
+  ///     List<Widget> previousChildren,
+  ///     bool reverse
+  ///   ) => Column(
+  ///     children: <Widget>[
+  ///      if (currentChild != null && !reverse) currentChild,
+  ///      ...previousChildren,
+  ///      if (currentChild != null && reverse) currentChild,
+  ///     ],
+  ///   ),
+  /// ),
+  /// ```
+  /// See also:
+  ///
+  ///  * [PageTransitionSwitcherLayoutBuilder] for more information about
+  ///    how a layout builder should function.
+  final PageTransitionSwitcherLayoutBuilder layoutBuilder;
+
+  /// The default layout builder for [PageTransitionSwitcher].
+  ///
+  /// This function is the default way for how the new and old child widgets are placed
+  /// during the transition between the two widgets. All children are placed in a
+  /// [Stack] that sizes itself to match the largest of the child or a previous child.
+  /// The children are centered on each other.
+  ///
+  /// See [PageTransitionSwitcherTransitionBuilder] for more information on the function
+  /// signature.
+  static Widget defaultLayoutBuilder(
+      Widget currentChild, List<Widget> previousChildren, bool reverse) {
```

Comment:
nit: 
```suggestion
      Widget currentChild, List<Widget> previousChildren, bool reverse,) {
```

... and then format with `flutter format`